src/utils/forms/commonValidatorRegler.ts

```diff
@@ -10,6 +10,14 @@ export function lagPostnummerRegel<T>(key: keyof T, getPostnummer: (t: T) => str
     };
 }
 
+export function lagPostboksnummerRegel<T>(key: keyof T, getPostboksnummer: (t: T) => string ): Valideringsregel<T>  {
+    return {
+        felt: key,
+        feilmelding: 'Postboksnummer kan maks ha 4 tall',
+        validator: (obj: T) => erTall(getPostboksnummer(obj)) && getPostboksnummer(obj).length < 5
```

Comment:
Vil denne regelen strippe bort whitespace? Ditto med tjenesten?

Reply:
nei, whitespace er det TPS som legger på for oss 😿 

Jeg har en egen PR som er merget i backend som trimmer postboksnummer før det returneres til frontend.